src/Layout/Presentation/DistrictOverview/DistrictOverview.tsx

```diff
@@ -27,7 +28,9 @@ export class DistrictOverview extends React.Component<DistrictOverviewProps, {}>
         const mostWeightedDistrict = data.find((entry) => entry.votesPerSeat === highestVotingPower)!.name;
         const leastWeightedDistrict = data.find((entry) => entry.votesPerSeat === lowestVotingPower)!.name;
         const calculateVulnerable = isQuotientAlgorithm(this.props.algorithm);
-        const mostVulnerable = calculateVulnerable ? getMostVulnerableSeatByQuotient(data) : undefined;
+        const mostVulnerable = calculateVulnerable
+            ? getMostVulnerableSeatByQuotient(data, this.props.districtThreshold)
+            : void 0;
```

Comment:
...void 0? No.

Reply:
I can revert these back to undefined, but then we will have to disable the Codacy rule for not using undefined. (Codacy recommended using void 0 instead)

Reply:
I'm still not sure why void 0 or undefined are even necessary. I don't think there's any benefit to passing void 0 over passing undefined or null. It's just confusing. What does the code do when it's undefined or void 0 or null?